frontend/packages/dev-console/src/components/edit-application/__tests__/edit-application-data.ts

```diff
@@ -437,6 +438,80 @@ export const gitImportInitialValues: GitImportFormData = {
     triggers: { webhook: true, image: true, config: true },
     strategy: 'Source',
   },
+  healthChecks: {
+    readinessProbe: {
```

Comment:
Why are you duplicating so much of data in this file? You can easily define one `const healthChecks: {...whateverProbes}` and then use it everywhere. Even in the resources object, you don't need to create a new resource object that has healthChecks entirely by copy pasting the whole thing again, You can just add new properties to the already present objects or clone it and add new properties for healthChecks. 

https://thevaluable.dev/dry-principle-cost-benefit-example/ might be useful.